news/2019-09-07-community-mentorship-program-fall-2020-mentee-signups-now-open.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: "Community Mentorship Program Fall 2020 Signups Now Open"
+date: 2019-09-08 12:50:00 +0000
+---
+
+The next cycle of the Community Mentorship program is starting again! If you've ever found yourself wanting to get into beatmapping, or expanding your current knowledge and skills, this program will serve as a great opportunity for you!
+
+![](/wiki/shared/news/banners/community-mentorship-program.jpg)
+
+The Community Mentorship program serves as place for upcoming mappers and modders to develop their skills underneath the guidance of more experienced mappers and modders in the community. Over the course of a cycle, you'll have the opportunity to learn from your mentor, participate in various mentorship events, such as mapping contests, hang out with fellow mentees and more!
+
+For those of you already experienced in mapping, and have a desire to take those that are newer under your wing, mentor applications are already open! In the forum announcement and rule thread [here](https://osu.ppy.sh/community/forums/topics/955356), you'll be able to apply if you’re considering mentoring! Keep in mind mentor applications take sometime to process, so please be patient.
```

Comment:
```suggestion
For those of you who are already experienced in mapping and have a desire to take those that are newer under your wing, mentor applications are open! In the [forum announcement and rule thread](https://osu.ppy.sh/community/forums/topics/955356), you can find more details about how to apply. Please keep in mind that mentor applications take some time to process, so you'll need to be patient.
```

Reply:
well according to this post they're already closed, so you'll need to rewrite or remove this. not sure why the news post is being made so late

Reply:
mhm yeah edited to reflect closed mentor apps